docs/administration/Configuration.md

```diff
@@ -564,6 +564,15 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - **Default**: 86400
 - **Description**: The timeout duration for the schema change operation (ALTER TABLE). Unit: seconds.
 
+#### fast_schema_evolution
+
+- **Default**: TRUE
+- **Description**: Whether to enable the fast schema evolution for all tables within the StarRocks cluster. Valid values are `TRUE` (default) or `FALSE`. Enabling this feature can increase the speed of schema changes and reduce resource usage when columns are added or dropped.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ac9c2d0a-74ed-4157-a8b6-8e9ef3753b99)
or -> and